### ISSUE TITLE
Opus: please support setting the header's output gain

### ISSUE THREAD

**Issue by ftrebien** [Moonbase59/loudgain#16]
[Opus developers used to recommend setting the header output gain to album gain](https://wiki.xiph.org/OggOpus#Comment_Header), [opusenc sets it to the value of REPLAYGAIN_ALBUM_GAIN when converting from FLAC](https://github.com/xiph/opus-tools/blob/master/src/flac.c#L179), [foobar2000 can set it](https://hydrogenaud.io/index.php?topic=113742.0) to album gain, track gain, or null, and defaults to album gain.

Therefore, I would like to kindly request an option to set the header output gain like foobar2000 does, or at least support setting it to album gain because it is usually a [it is usually a better normalization mode](https://productionadvice.co.uk/tidal-normalization-upgrade/).

**Comment by Moonbase59** [comment]
You’re right, and I’d much like to be able to write the header, but TagLib (which loudgain relies on) doesn’t support that (yet).

You might want to support https://github.com/taglib/taglib/issues/916 – thanks!

**Comment by ftrebien** [comment]
I actually saw your issue there and noticed there was no reply from developers yet. I suspect that TagLib may not be interested in adding such support because, well, pedantic details: headers are not tags. r128gain has [solved](https://github.com/desbma/r128gain/blob/master/r128gain/opusgain.py) this problem (though it's still considered an "experimental" feature) by writing to the header directly instead of delegating that task to its tagging library (Mutagen).
